docs/sc_FunctionalRequirements.csv

```diff
@@ -8,114 +8,114 @@ d)       ATS Command Error Counter
-"
-SC2000,SC2000,SC shall define <PLATFORM_DEFINED> bytes of storage for each of two (2) Absolute Time-tagged command Sequence (ATS) tables.,"One table will be used for uploads, while the other is executing."
-SC2000.1,SC2000.1,SC shall resolve time to 1 second for Absolute Time Command Sequences (ATS).,Heritage missions have shown that this is adequate resolution.
-SC2000.2,SC2000.2,SC shall accept a maximum of <PLATFORM_DEFINED> commands per ATS.,Required in order to bound the command number.
+"
+SC2000,SC2000,SC shall define <PLATFORM_DEFINED> bytes of storage for each of two (2) Absolute Time-tagged command Sequence (ATS) tables.,"One table will be used for uploads, while the other is executing."
+SC2000.1,SC2000.1,SC shall resolve time to 1 second for Absolute Time Command Sequences (ATS).,Heritage missions have shown that this is adequate resolution.
+SC2000.2,SC2000.2,SC shall accept a maximum of <PLATFORM_DEFINED> commands per ATS.,Required in order to bound the command number.
 SC2000.3,SC2000.3,"SC shall accept a variable number of variable length commands from each ATS. Each ATS command shall contain:
 a)       A command number
 b)       A time tag denoting the time at which to execute the command
-c)       A variable length command",
+c)       A variable length command",
 SC2000.4,SC2000.4,SC shall use the <PLATFORM_DEFINED> time format (UTC or TAI).,"Provides flexibility on time format.
-"
+"
 SC2001,SC2001,"Upon receipt of a table update indication for an ATS Table, SC shall validate the following for the ATS table:
 a)       Duplicate command numbers
 b)       Invalid command lengths
 c)       Commands that run off of the end of the table
-d)       Command number",SC application needs to validate the table to ensure gross errors are caught.  cFE Table Services handles the ground interface to tables. 
-SC2002,SC2002,SC shall allocate <PLATFORM_DEFINED> Relative Time-tagged Sequences (RTSs) with each capable of storing <PLATFORM_DEFINED>  bytes of stored command data.,Specifies the limits for table sizing purposes.
-SC2002.1,SC2002.1,SC shall resolve time to 1 second for Relative Time Command Sequences (RTS).,Heritage missions have shown that this is adequate resolution.
-SC2002.2,SC2002.2,SC shall accept variable length packed RTS commands within the <PLATFORM_DEFINED> byte relative time-tagged sequences.,
+d)       Command number",SC application needs to validate the table to ensure gross errors are caught.  cFE Table Services handles the ground interface to tables. 
+SC2002,SC2002,SC shall allocate <PLATFORM_DEFINED> Relative Time-tagged Sequences (RTSs) with each capable of storing <PLATFORM_DEFINED>  bytes of stored command data.,Specifies the limits for table sizing purposes.
+SC2002.1,SC2002.1,SC shall resolve time to a wakeup count for Relative Time Command Sequences (RTS).,To address the bug where RTS commands could be delayed when the 1 Hz wakeup occurs near a second rollover.

```

Comment:
Typically in requirements we wouldn't specify the "why," just the "what," In this case it isn't resolving time at all anymore. I would word this to say

```
SC shall track a wakeup count for Relative Time Command Sequences (RTS)
```